src/components/ebay-listbox-button/index.marko

```diff
@@ -33,9 +35,15 @@ $ var selectedText = selectedOption && selectedOption.text;
         disabled=input.disabled
         name=input.buttonName
         aria-haspopup="listbox"
+        aria-labelledby=(labelId && `${input.prefixId} ${labelId}`)
```

Comment:
Is this intentional? I'm assuming this should just be `aria-labelledby=labelId`.

Reply:
Yes it is intentional, the `labelledby` needs to be:
`externalLabelID internalLabelId`
So if the `input.prefixId` is present, then we will add a `labelId` to our internal label and then combine both. 
See: https://github.com/eBay/ebayui-core/issues/1321